### DISCUSSION TITLE
Pi5 testing

### DISCUSSION THREAD

**Discussion by Exarkuniv** [Exarkuniv/RetroPie-Extra#182] [General]
List here the ones that are working. Also if you want to list what you found to get others working or anything else that you did to make a script working. 

you dont have to go that far. just testing them is a big help

**Comment by RapidEdwin08** [comment]
List of Extras I've tested with _No Issues_ on RPi5 Bookworm 64bit:

> rott-darkwar, rott-huntbgin (Both 0lder rott-* + Newer rott-expr*).
> pcexhumed

List of Extras I've tested that _Needed Dependency Updates_ to resolve compile issues on RPi5 Bookworm OS 64bit for:

> chocolate-doom:  python-pil  -> python3-pil
> crispy-doom:  python-pil  -> python3-pil
> dunelegacy:  libfluidsynth1 -> libfluidsynth3 (duneii data also in wrong dir, needs ../ports/dune2/data/*).

I also had to add 2 more _lazy_ sed fixes for dunelegacy related to Bookworm OS, + used _make -j4_, and set _Fullscreen = false_ in the INI after:
```
# Bookworm error: field ‘mentatStrings’ has incomplete type ‘std::array<std::unique_ptr<MentatTextFile>, 3>’
sed -i '1s/^/#include <tuple> \n/' $md_build/src/FileClasses/TextManager.cpp
sed -i '1s/^/#include <array> \n/' $md_build/src/FileClasses/TextManager.cpp
	
# Flickering fix -> Comment 1st instance of  // SDL_RenderPresent(renderer);
sed '0,/SDL_RenderPresent(renderer);/s//\/\/SDL_RenderPresent(renderer);/' $md_build/src/Game.cpp > /dev/shm/Game.cpp; sudo mv /dev/shm/Game.cpp $md_build/src/Game.cpp
```

0ther Extras that compiled OK but had _Issues_ running on RPi5 Bookworm 64bit:

> rednukem (Fails to open with xinit: connection to X server lost)
> I ended up Compiling manually from nukeykt/NRedneck.git:
> make LTO=0 SDL_TARGET=2 STARTUP_WINDOW=0
> Then set Polymer = 0 + ScreenMode = 0 in the rednukem.cfg

> hexen2 (GL) Scrambled Video issues
> I've found that you can adjust the hexen config.cfg setting file to find a Working Video Resolution, +
> you need to match it in RetroPie as well in /opt/retropie/configs/all/videomodes.cfg.
> Simply toggling Fullscreen 0ff sometimes works for 0ther Emulators/Ports with Scrambled Video issues as well.
> Hoping Scrambled Video Issues will eventually be resolved once we get Legit Video Drivers for Pi5...

I also created an _aethersx2_ script in my fork of Retropie-Setup.
It's hardly clean, and it's pulling it from the wayback archives, but it works for now.
You are more than welcome to it, and the corresponding assets.

Thanks for all you do maintaining Extras.

**Reply by RapidEdwin08** [reply]
I saw you added aethersx2, but I don't see libfuse2 in the depends. Not sure if still needed, but it was a requirement at some point.
getDepends libfuse2 ...

**Reply by Exarkuniv** [reply]
i have tested it for a week getting it to work right on a fresh build image with Pi OS lite and the latest pull from retropie github

everytime i run the script and start GTA3 it works just fine for me. so unless its needed for something else like game saves or something its not needed that i know of

**Reply by RapidEdwin08** [reply]
It's required if using the AppImage.
But I just noticed you guys extracted it. 
So it's not needed in this case.
Great addition to Extras.

**Comment by Giandj** [comment]
HI EXARKUNIV YOU WROTE:
 - lr-samecdi - Philips CDI - same_cdi port for libretro - Installs, Dont have games to test

I HAVE TESTED SOME GAMES FOR YOU, AND WORKS GREAT (YOU NEED TO SPECIFY BIOS), I have tested it on Reropie v4.8.8  and pi5 4Gb.

N.3 Bios files (for Philips CD-I) are needed, (cdibios.zip, cdimono1.zip, cdimono2.zip ), you must place this 3 files under 
this directory:
/home/pi/RetroPie/BIOS/same_cdi/bios/

CHECK EVERY .ZIP BIOS FILES ON "SAME-CDI GITHUB MANUALS", BECAUSE SOME BIOS FILES NEEDED ARE MISSING FROM THE 3 .ZIP FILES...

Put some rom games in:
 /home/pi/RetroPie/ROMS/cdimono1/

And IT works great !

**Reply by Exarkuniv** [reply]
Thanks for checking.

I'll add the files to the help part and also the GitHub page for more information.
